srcpkgs/litterbox/template

```diff
@@ -11,4 +11,4 @@ maintainer="Allen Sobot <chilledfrogs@disroot.org>"
 license="GPL-3.0-or-later"
 homepage="https://git.causal.agency/litterbox/"
```

Comment:
```suggestion
homepage="https://git.causal.agency/litterbox/"
changelog="https://git.causal.agency/litterbox/tag/?h=${version}"
```

Reply:
Wasn't sure that'd be considered an appropriate changelog, thanks, done

Reply:
well, better something than nothing